components/miller/stories/miller.stories.js

```diff
@@ -2,24 +2,13 @@
 import { Template } from "./template";
 
 export default {
-  title: "Miller",
-  description: "The Miller component is...",
+  title: "Miller Columns",
+  description: "Miller columns are a browsing/visualization technique that can be applied to tree structures. The columns allow for multiple levels of the hierarchy to be open at once and provide a visual representation of the current location.",
   component: "Miller",
   argTypes: {
```

Comment:
Might want to add: `columns: { table: { disable: true } }` so that the raw objects don't show up in the controls section